CON-68: Replace various occurrences of "consent management" (#38)
GovStackWorkingGroup/bb-consent: spec/4-key-digital-functionalities.md

The Consent Building Block enables organisations to enforce Data Policies that require signed consent by Individuals for the use of their personal data. Its key purpose is to allow individuals to view Consent Agreements and sign or withdraw their consent on what personal data is used and accessible to organisations. It also clarifies the Data Policy applied, such as the purpose, retention period, jurisdiction, third-party data sharing, etc.

> The Consent Building Block implements the key functionalities described in the [consent lifecycle](2-description.md#23-consent-agreement-lifecycle). It includes the ability to configure consent agreements by an organisation admin, present consent requests towards individuals, capture consents, enable queries if consent exists, or not, and enable independent audit of consents.

Reviewer: ```suggestion
The Consent Building Block implements the key functionalities described in the [consent agreement lifecycle](2-description.md#23-consent-agreement-lifecycle). It includes the ability to configure consent agreements with reference to relevant data policy, present consent requests towards individuals, capture consents, enable queries if consent exists, or not, and enable independent audit of consents.
```

> The functionalities are derived from the [consent agreement lifecycle](2-description.md#23-consent-agreement-lifecycle) and categorised according to the [Actors](2-description.md#2.4-actors) described above. While the consenting workflows (as described above) are implicitly considered the centrepiece of the Consent Building Block, it is important to realise that the integrity of consent data can only be achieved if robust configuration before and auditing after the Consent Agreement signing and Consent Record verification activities are in place. Hence, the functionalities are described following the logical sequence of the consent agreement lifecycle and they are all equally important components of the Consent Building Block.

Reviewer: ```suggestion
The functionalities are derived from the [consent agreement lifecycle](2-description.md#23-consent-agreement-lifecycle) and categorised according to the [Actors](2-description.md#2.4-actors) described above. While the consenting workflows (as described above) are implicitly considered the centrepiece of the Consent Building Block, it is important to realise that the integrity of consent data and related process can only be achieved if robust configuration before and auditing after the Consent Agreement signing and Consent Record verification activities are in place. Hence, the functionalities are described following the logical sequence of the consent agreement lifecycle and they are all equally important components of the Consent Building Block.
```

### 4.1 Administrator User Functionalities

> The Administrator (Data Provider or Data Consumer Admin) configures the Consent Building Block on behalf of the organisation. For simplicity, it is foreseen that one organisation involved in the data processing transaction (that is either Data Provider or Data Consumer) takes the responsibility for the configuration of the Data Policy and respective Consent Agreements(s), and so that organisation’s Administrator maintains the required configurations.

Reviewer: ```suggestion
A Configuration Administrator (Data Provider or Data Consumer Admin) configures the Consent Building Block on behalf of the organisation. For simplicity, it is foreseen that one organisation involved in the data processing transaction (that is either Data Provider or Data Consumer) takes the responsibility for the configuration of the Data Policy and respective Consent Agreements(s), and so that organisation’s Administrator maintains the required configurations.
```

The table below summarises the key use cases identified for an organisation's Administrator. Organisations can be Data Consumers or Data Providers, i.e. the organisations legally delegated the responsibility for collecting consent for the systems handling personal data processing.

> | **Consent use-cases**                                                                                                                                               | **Link(s) to the UCS**                                                                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| **Consent lifecycle use-cases**                                                                                                                                               | **Link(s) to the UCS**                                                                                                                                                                                                                                                                                    |
```

The table below summarises the key use cases identified for the Individuals.

> | **Consent use-cases**                                                                                                                                                                                             | **Link to the UCS**                                                                                                                              |

Reviewer: ```suggestion
| **Consent lifecycle use-cases**                                                                                                                                                                                             | **Link to the UCS**                                                                                                                              |
```

**Also to consider:** “READ CONSENT STATUS” use-case is also used by any workflow (and Actor) that requires verification of the consent status (for example, before executing the data transfer from Data Providing System to the Data Consuming System).

> | **Consent use-cases**                                                                                    | **Link to the UCS**                                                                                                                                                                 |

Reviewer: ```suggestion
| **Consent lifeycle use-cases**                                                                                    | **Link to the UCS**                                                                                                                                                                 |
```
